README.md

```diff
@@ -20,3 +20,5 @@ to be the most efficient one!
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+Beginning with the initialization step, we note that initializing the 'visited' distances arrays takes O(V) time where V represents the # of nodes in the graph. Pressing forward we see that the outer loop runs for V-1 iterations which takes O(V) time again. Of course, for each outer loop iteration we also have a nested inner-loop that will iterate over all the vertices. Lastly, we have one more nested loop that iterates over all of the vertices and edges. Putting it all together we get a big $\Theta$ complexity of $(O(V) + (O(V) * (O(V)) + O(V)) + O(E)) = (O(V^2) + 2 O(V)) + O(E) => \theta (V^2) + (E)$
```

Comment:
Could you do this more efficiently? Looking for the minimum distance node at each iteration could be optimized e.g. with a priority queue.

Reply:
Would you like me to describe what this optimization would look like in the markdown only or do you also want to see an attempt made to utilize a priority queue in my code.js file?

Reply:
You could use a priority queue, or something else that makes this step more efficient. Going through the entire list of nodes each time is unnecessarily inefficient.